packages/@okta/vuepress-site/docs/api/resources/apps/index.md

```diff
@@ -3564,7 +3564,7 @@ Assigns a group to an application
 | Parameter     | Description                                     | Param Type | DataType                                      | Required | Default |
 | ---------     | ----------------------------------------------- | ---------- | --------------------------------------------- | -------- | ------- |
 | applicationId | `id` of an [app](#application-model)            | URL        | String                                        | TRUE     |         |
-| groupId       | unique key of a valid [Group](groups)           | URL        | String                                        | TRUE     |         |
+| groupId       | unique key of a valid [Group](/docs/api/resources/groups)           | URL        | String                                        | TRUE     |         |
```

Comment:
Shouldn't this be `/docs/api/resources/groups/` (w/trailing slash)?  The site corrects it, but...

Reply:
I don't have an opinion on this one.  We could lint it, but that feels a bit over-bearing.

Reply:
When we get a good winter set up, I'll add this and fix everything that's broken

Reply:
By broken, I mean slash missing